README.md

```diff
@@ -13,15 +13,15 @@
         <img src="https://img.shields.io/badge/Fivetran_Quickstart_Compatible%3F-yes-green.svg" /></a>
 </p>
  
-# Qualtrics Source dbt Package ([Docs](https://fivetran.github.io/dbt_qualtrics_source/))
+# Qualtrics Source dbt Package ([Docs](https://fivetran.github.io/dbt_qualtrics_source/)) 
 # 📣 What does this dbt package do?
 <!--section="qualtrics_source_model"-->
 - Materializes [Qualtrics staging tables](https://fivetran.github.io/dbt_qualtrics_source/#!/overview/github_source/models/?g_v=1) which leverage data in the format described by [this ERD](https://fivetran.com/docs/applications/qualtrics/#schemainformation). These staging tables clean, test, and prepare your Qualtrics data from [Fivetran's connector](https://fivetran.com/docs/applications/qualtrics) for analysis by doing the following:
   - Name columns for consistency across all packages and for easier analysis
     - Primary keys are renamed from `id` to `<table name>_id`. 
     - Foreign key names explicitly map onto their related tables (ie `owner_id` -> `owner_user_id`).
     - Datetime fields are renamed to `<event happened>_at`.
-  - Adds freshness tests to source data
+  - Adds freshness tests to source data 
```

Comment:
On a similar note, we should remove this freshness test call out if we are not testing freshness.

Reply:
I appreciate how you handled this by not just removing it and instead calling attention to _why_ the freshness tests are removed. Great job!